### DISCUSSION TITLE
github self hosted runners unattended installation

### DISCUSSION THREAD

**Discussion by paulpuvi06** [community/community#27014] [Actions]
<p>Hello Experts, </p>
<p> Do we have possiblity for unattended installation of self hosted runner ?</p>
<p> Is there a way to check action minutes and switch to self hosted based on condition in action template? </p>
<p>Thanks in advance.</p>
<p>Regards,<br>
Paul</p>

**Comment by paulpuvi06** [comment]
<ul>
<li>I am able to do unattened install after adding <strong>–unattended</strong> flag, this is not mentioned in <a href="https://help.github.com/en/actions/hosting-your-own-runners" rel="nofollow noopener">https://help.github.com/en/actions/hosting-your-own-runners</a> . it would be good to have instructions on  <strong>unattended installation.</strong>
</li>
</ul>
<p>do we have any answers for my second question ? </p>

**Comment by Yanjingzhu** [comment]
<p>Hi <a class="mention" href="/paulpuvi06">@paulpuvi06</a> ,</p>
<p>For your second question, I am afraid that the answer is no. </p>
<p>You could check the runner minutes quota in Billing page: <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/viewing-your-github-actions-usage" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/viewing-your-github-actions-usage</a></p>
<p>But you need to manually change the runs-on key value in your workflow yml file to switch hosted runner to self hosted runner.   </p>

**Comment by paulpuvi06** [comment]
<p>Hi <a class="mention" href="/yanjingzhu">@yanjingzhu</a>  , </p>
<p>Thank you. is there a plan to add this feature in github action flow or to introduce REST api to check the status and switch to self hosted agnet after 50000 minutes. </p>
<p>Regards,<br>
Paul</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Currently, this is not a plan for your requirement. I would encourage you sharing your idea in the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">Feedback form for GitHub Actions</a>. Thank you for helping us build a better GitHub Actions.</p>
